metricbeat/docs/modules/kubernetes.asciidoc

```diff
@@ -181,6 +181,8 @@ Kubernetes cluster overview example:
 
 image::./images/metricbeat-kubernetes-clusteroverview.png[]
 
+If you are setting collection period to a value bigger than `2m` you will need to increase
```

Comment:
The 2 minutes where just a placeholder - is this a good time range? What's the default collection period?

Reply:
Agreed, if "normal" periods are 10s - 1m, then 2m is a good tradeoff